roles/openshift_logging/tasks/label_node.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I worry that label_node.yml might conflict with other tools that are managing node labels. @abutcher what are your thoughts?